### PR TITLE
guide for tracer agent host change

### PR DESCRIPTION
**Pull request by dussault-antoine**
Agent vs. Tracer host public facing guide, to explain in more details what is the difference between the Datadog Agent host and the Tracer host

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by maycmlee** [comment]
Created [DOCS-6657](https://datadoghq.atlassian.net/browse/DOCS-6657) for docs review.

File: content/en/tracing/guide/agent_tracer_hostnames.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{{< img src="/tracing/guide/agent_tracer_hostnames/remote_host.png" alt="Agent deployed on a remote host, different from the application" style="width:80%;" >}}
+
+### When are the Tracer and Agent hosts set on spans ?
+
+- The Datadog Agent hostname is always set on spans.
+- The Tracer hostname is set on spans iif `DD_TRACE_REPORT_HOSTNAME` is set to `true` (default `false`).
+
+## When is the host information used in APM ? 
+
+When creating [retention filters][2], [metrics from spans][3] or [sensitive data scanner rules][4] using host tag filters (e.g. `availbility-zone`, `cluster-name`) in queries, these host tags are enriched from the **Datadog Agent host** information.
```
Comment (brett0000FF) [inline content/en/tracing/guide/agent_tracer_hostnames.md:31]:
```suggestion
### When does APM use host information? 

APM uses host information when you create [retention filters][2], generate [metrics from spans][3], or create [sensitive data scanner rules][4] using host tag filters in queries. For example, host tag filters like `availability-zone` and `cluster-name` are enriched from the Datadog Agent host information.
```
The original sentence was a little unclear to me. Let me know if I misinterpreted it!

File: content/en/tracing/guide/agent_tracer_hostnames.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: Understand Datadog retention policy to efficiently retain trace data
+kind: guide
+---
+
+## Overview
+
+When using Datadog APM, the `host` tag is used to correlate spans and traces to Infrastructure Monitoring data, in order to see host metrics associated with the host from the span/trace.
+
+### Datadog Agent vs. Tracer hostname
```
Comment (brett0000FF) [inline content/en/tracing/guide/agent_tracer_hostnames.md:10]:
```suggestion
## Datadog Agent vs. Tracer hostname
```

**Review by brett0000FF — APPROVED** [review]
Looks good! I added a commit to include a link to this page on the guides page, and I offered a few suggestions for you to consider.

**Pull request merged**